project.clj

```diff
@@ -11,6 +11,8 @@
                  [org.clojure/clojurescript "1.7.122"]
                  [org.clojure/core.async "0.2.374"]]
 
+  :plugins [[lein-cljfmt "0.3.0"]]
```

Comment:
Yes. This plugin will be available in all the profiles, whereas putting it there would mean it would only be available under the dev profile. There isn't much of an effective difference in this case.